hockey/sortFunctions.php

```diff
@@ -0,0 +1,60 @@
+<?php
+function getInfoSort($column,$how,$num){
+    $sort = dbQuery("
+        SELECT *
+        FROM hockey_1617
+        ORDER BY $column $how, LastName
+        LIMIT $num,50");
+    return $sort->fetchAll();
+}
+function tableContentsSort($column,$how,$num,$end=50){
+    $info = getInfoSort($column,$how,$num);
+    $result = '';
+    for ($i=0,$j=$num+1;$i<$end;$i++,$j++){
+        $result .= '
+        <tr>
+            <td class="numbers">'.$j.'</td>
```

Comment:
This works, but you don't really need to keep track of `$j` separately. I think you could just leave `$j` out entirely and put `$i+$num+1` here.